src/vm/jitinterface.cpp

```diff
@@ -6033,8 +6033,8 @@ CorInfoHelpFunc CEEInfo::getNewHelperStatic(MethodTable * pMT, bool * pHasSideEf
 
     // Slow helper is the default
     CorInfoHelpFunc helper = CORINFO_HELP_NEWFAST;
-    bool hasFinalizer = pMT->HasFinalizer();
-    bool isComObjectType = pMT->IsComObjectType();
+    bool hasFinalizer = pMT->HasFinalizer() != 0;
```

Comment:
returns `DWORD`

Reply:
The MethodTable methods use `DWORD` and `BOOL` interchangeably. Changing this to `BOOL hasFinalizer = pMT->HasFinalizer()` would work too and it would look tiny bit better (symmetric with the next line).

Reply:
That then gives a similar error for the assignment: `*pHasSideEffects = hasFinalizer;`?

Reply:
Changed to `*pHasSideEffects = (hasFinalizer != 0)`